### DISCUSSION TITLE
Questions about creating a new PDK

### DISCUSSION THREAD

**Discussion by ilryeong** [The-OpenROAD-Project/OpenROAD-flow-scripts#2522] [Q&A] [answered]
It might be awkward because you use the translator
Please keep that in mind and read it

I'm trying to create a new platform using a new PDK instead of the platform supported by openload
So I made it to Synthesis
However, it was blocked in the floorplan stage.
If you look at config.mk on the platform, PLACE_SITE is at the front of the floorplan stage, what does this mean?
I wrote config.mk referring to nangate45, but I don't have any information about PLACE_SITE.
The problem is that if I make, I cannot find PLACE_SITE in floorplan stage and proceed.
If you know about this, please let me know what to do

Thank you for reading so far

**Comment by maliberty** [comment, marked as answer]
PLACE_SITE is the SITE from your LEF that is used to build the placement rows in initialize_floorplan.

**Reply by ilryeong** [reply]
I get it for now.
But why are you looking for PLACE_SITE when the lef file is in the lef folder of platforms?

I downloaded Openroad from Ubuntu to git.
But it doesn't seem to say where to find PLACE_SITE among the downed ones
So how do I find PLACE_SITE?

**Reply by maliberty** [reply]
There can be many sites in the LEF and the user can control what is used.

**Reply by ilryeong** [reply]
I knew what to write for PLACE_SITE.
Thank You.
